docs/apache-airflow-providers-dbt-cloud/operators.rst

```diff
@@ -69,6 +69,14 @@ referenced within the ``default_args`` of the example DAG.
     :start-after: [START howto_operator_dbt_cloud_run_job]
     :end-before: [END howto_operator_dbt_cloud_run_job]
 
+Also you can use deferrable mode in this operator if you would like to free up the worker slots while the sensor is running.
```

Comment:
```suggestion
Also you can use deferrable mode in this sensor if you would like to free up the worker slots while the sensor is running.
```
Also, this should live in the sensor section rather than the DbtCloudRunJobOperator section.